.github/workflows/test.yml

```diff
@@ -48,3 +48,31 @@ jobs:
 
       - run: |
           yarn run test:ci
+
+      - uses: actions/upload-artifact@v3
+        if: github.ref == format('refs/heads/{0}', github.event.repository.default_branch)
+        with:
+          name: base-coverage-summary
+          path: ./coverage/coverage-summary.json
+
+      - uses: dawidd6/action-download-artifact@v2
+        if: github.ref != format('refs/heads/{0}', github.event.repository.default_branch) && github.event_name == 'pull_request'
+        with:
+          branch: ${{ github.event.repository.default_branch }}
+          name: base-coverage-summary
+          path: ./coverage/base
+          workflow: test.yml
+
+      - uses: adRise/jest-cov-reporter@main
```

Comment:
this is a private action? I searched 0 result in Github Actions Marketplace
https://github.com/marketplace?type=actions&query=adRise%2Fjest-cov-reporter+

Reply:
you'll have to search by another name instead
<img width="792" alt="image" src="https://user-images.githubusercontent.com/5073227/194747621-7de01ac4-77bc-4871-a1a1-942d6bdec148.png">

Reply:
but I'll use other coverage reporter instead, this one is for `jest`, while we are using `nyc + mocha`.